### PR TITLE
confine scroll in assignments question sidebar

### PR DESCRIPTION
**Pull request by SamamthaH**
## Description

Describe your changes in detail
Added a tui scroll bar around the list of questions to have a nicer functioning scrollbar and hide the scrollbar when it is not needed, and I updated the width of the questions to fill the width of the sidebar, it is fixed because the sidebar size never changes.

## Types of changes

What types of changes does your code introduce?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (only improve the code quality no change in functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] Issue has been created
- [x] Every file touched is linted
- [ ] Every file touched has tests that cover all the funcitonality with green coverage
- [ ] Documentation is updated (if applicable).

## Issue

Please link the issue here.
https://github.com/canvas-gamification/canvas-gamification-ui/issues/690

## Tests

Please describe in detail what tests you added or changed.

## Screenshots (if appropriate):
image
image
image

### CODE REVIEW

File: src/app/course/event/event-question-view/event-question-view.component.scss [hunk 1]
```diff
@@ -24,3 +24,11 @@
...
     }
 }
+
+.box {
+    height: calc(80vh - 12rem - 2px); // Window height - (header + footer + wiggle room) - border width
+}
+
+.fill-space {
+    width: 22vh;
+}
```
Comment (keyvankhademi) [inline src/app/course/event/event-question-view/event-question-view.component.scss:34]: have you used this?

File: src/app/course/course.module.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ import {
     TuiLoaderModule,
     TuiModeModule,
     TuiNotificationModule,
-    TuiPointerHintModule,
+    TuiPointerHintModule, TuiScrollbarModule,
```
Comment (keyvankhademi) [inline src/app/course/course.module.ts:63]: please format this each module in 1 line

File: src/app/course/course.module.ts [hunk 3]
```diff
@@ -201,15 +201,16 @@ import {ConceptListComponent} from './concept-list/concept-list.component'
...
         TuiRingChartModule,
+        TuiScrollbarModule,
         TuiSelectModule,
         TuiSidebarModule,
         TuiStepperModule,
         TuiSvgModule,
         TuiTableModule,
         TuiTabsModule,
         TuiTagModule,
-        TuiTextAreaModule,
```
Comment (keyvankhademi) [inline src/app/course/course.module.ts:211]: try to move random stuff around

Comment (SamamthaH) [reply src/app/course/course.module.ts:211]: Honestly the linting just did that as I was fixing the placement of the scrollbar import but I can move it back

File: src/app/course/event/event-question-view/event-question-view.component.scss (hunk 1 above)
Comment (SamamthaH) [reply src/app/course/event/event-question-view/event-question-view.component.scss:34]: Line 43 of event-question-view.component.html, it is what causes the question name to push the open/close button into its proper place

**Review by keyvankhademi — APPROVED** [review]

**Pull request merged**